package cms_8120;
import "type/cmd.proto";
import "type/ModuleType.proto";

//客户端退出消息
message MsgClientLogout
{
    optional int32 cmd                  = 1;    // CMD_MSG_CLIENT_LOGOUT
    optional int32 moudle_type          = 2;    //退出模块类型
    optional string user_name           = 3;    //用户账户
    optional string ip                  = 4;    // 客户端ip
    optional string mac                 = 5;    // 客户端mac
    optional string client_version      = 6;    // 客户端版本
    optional string dc_ip               = 7;    //数据中心服务IP
}
